pkg/webhook/wip.go

```diff
@@ -0,0 +1,82 @@
+// Copyright © 2017 Syndesis Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package webhook
+
+import (
+	"net/http"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-github/github"
+	"github.com/pkg/errors"
+)
+
+const (
+	wipLabel        = "wip"
+	doNotMergeLabel = "do not merge"
+
+	wipContext = "wip"
+)
+
+var (
+	wipHandler Handler = &wip{}
+
+	wipRE, _ = regexp.Compile(`\b(?:` + wipLabel + `|` + doNotMergeLabel + `)\b`)
```

Comment:
Wouldn't it be better to match with case ignored ? (people often use a "WIP" in uppercase in the title)

Reply:
Good spot - updated.